faq.html

```diff
@@ -321,7 +321,7 @@ <h2>Miscellaneous</h2>
 <p>Bitcoin can be purchased via many channels: friends or family, Bitcoin meetups, Bitcoin ATMs, Vouchers, Localbitcoins or from any of the centralized exchanges.</p>
 
 <p><a name="15"></a><strong>Does Bisq support ‘XYZ’ payments?</strong></p>
-<p>Bisq will support a wide range of payment methods for national currencies. The criteria for inclusion are that the payment method is safe against chargebacks, and that it allows the arbitrator to view evidence of the transaction. National bank transfer is supported in every country (expect the US - but there is Zelle and others).</p>
+<p>Bisq will support a wide range of payment methods for national currencies. The criteria for inclusion are that the payment method is safe against chargebacks, and that it allows the arbitrator to view evidence of the transaction. National bank transfer is supported in every country (except the US - but there is Zelle and others).</p>
```

Comment:
Actually that text is outdated. We re-enabled national bank transfer in 0.6.6 for the US. Could you remove the text in the brackets and update the PR?

Reply:
Looks like you already committed the fixed text to your website/master branch, and I created an unfixable [so far] merge conflict on this file when I tried to sync my fork to yours.  

The commits to my branch below ( 8c7edf0 , 823a5c0 ) were unsuccessful attempts to fix the merge conflicts, but you do not need to pull/merge them anyway, and I am not sure what to do next.

Reply:
Looks like no further changes need be merged from my fork, and Chris Beams closed Issue #45 

See comment https://github.com/bisq-network/website/issues/45#issuecomment-369452486